x-pack/plugins/alerting/server/application/rule/methods/bulk_edit/schemas/bulk_edit_rules_option_schemas.ts

```diff
@@ -38,6 +39,15 @@ const bulkEditActionSchema = schema.object({
       notifyWhen: notifyWhenSchema,
     })
   ),
+  alertsFilter: schema.maybe(actionAlertsFilterSchema),
+  type: schema.literal(RuleActionTypes.DEFAULT),
+});
+
+export const bulkEditSystemActionSchema = schema.object({
```

Comment:
System actions are not allowed to contain: `group`, `alertsFilter`, and `frequency`.